app/controllers/mortgage_calculator/stamp_duties_controller.rb

```diff
@@ -1,22 +1,39 @@
 module MortgageCalculator
   class StampDutiesController < ::MortgageCalculator::ApplicationController
+    CALCULATOR = MortgageCalculator::StampDuty
+    before_action :set_rates
+
     def show
       @stamp_duty = StampDuty.new
-      @rates = MortgageCalculator::StampDuty::STANDARD_BANDS
+      @javascript_config = {
```

Comment:
@gotbadger if you intend to share variables between the _client_ and the _server_ then this gem [Gon](https://github.com/gazay/gon) will be perfect for that. 

## Setting Variables on Rails
```ruby
gon.rates = {
  standard: CALCULATOR::STANDARD_BANDS,
  ftb: CALCULATOR::FIRST_TIME_BUYER_BANDS,
  second_home_tax_rate: CALCULATOR::SECOND_HOME_ADDITIONAL_TAX,
  second_home_threshold: CALCULATOR::SECOND_HOME_THRESHOLD,
  first_time_buyer_threshold: CALCULATOR::FIRST_TIME_BUYER_THRESHOLD
}
```

## Using Variables in JS
```javascript
gon.rates
```

Reply:
Feel like this is probably overkill for this situation? It looks like this gem works essentially the same as this code is right now anyway.

Reply:
No problemo. I thought I'd let you know.

Reply:
I agree with @gotbadger to be overkill. BUT IMHO this variable should not be a controller instance variable and it appears to be to more a helper method containing this constants. And other thing, "javascript_config" sounds very generic to me. Can you rename to something which this config is for? Wdyt?